opencog/cogserver/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Step to take
+# 1. docker build --no-cache -t opencog/cogserver .
+# 2. docker run -p 17001:17001 -it opencog/cogserver
+# 3. rlwrap telnet localhost 17001
+
+FROM opencog/opencog-deps
+
+# Clone opencog and build it
+RUN git clone --depth 1 https://github.com/opencog/opencog oc
+RUN cd oc; mkdir build; cd build; cmake ..; make -j$(nproc)
```

Comment:
having the dependencies only in the base is nice for implementing shared filesystem with host, where we have our git clone and would help persist the data even on removing containers. This file is just for some one who wants to get up an running with opencog, mostely a user i assume.  But nothing prevents one from using this for development of course.